UPDATING.md

```diff
@@ -39,28 +39,36 @@ assists users migrating to a new version.
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Airflow Master
 
+### Changes to `aws_default` Connection's default region
+
+The region of Airflow's default connection to AWS (`aws_default`) has
+previously been set to `us-east-1` during installation.
+
+The region now needs to be set manually either in the connection screens in
+Airflow, or via the `AWS_DEFAULT_REGION` environment variable.
+
 ### Changes to FileSensor
 FileSensor is now takes a glob pattern, not just a filename. If the filename you are looking for has `*`, `?`, or `[` in it then you should replace these with `[*]`, `[?]`, and `[[]`.
 
 ### Change dag loading duration metric name
-Change DAG file loading duration metric from 
```

Comment:
Could you avoid making white-space only changes to the rest of this file? (This will make this change harder to cherry-pick back to the release branch)

Reply:
Understood -- will do in a new commit.

Reply:
These changes still are here - can you remove them please.